### PR TITLE
workflows: improve setup-homebrew usage.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
https://github.com/Homebrew/actions/pull/393 will change the default behaviour of setup-homebrew so let's be explicit to speed things up and avoid changes when it's merged

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -36,6 +36,10 @@ jobs:
       - name: Set up Homebrew
         id: set-up-homebrew
         uses: Homebrew/actions/setup-homebrew@master
+        with:
+          core: true
+          cask: false
+          test-bot: false
```
Comment (Bo98) [inline .github/workflows/tests.yml:43]: test-bot used here

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -59,6 +63,10 @@ jobs:
...
```
Comment (Bo98) [inline .github/workflows/tests.yml:69]: test-bot used here

**Review by Bo98 — COMMENTED** [review]
`core` and `cask` have no effect in this repository, so this PR won't really matter. But probably worth having anyway.

File: .github/workflows/automerge.yml [hunk 3]
```diff
@@ -187,6 +187,8 @@ jobs:
       - name: Set up Homebrew
         uses: Homebrew/actions/setup-homebrew@master
         with:
+          core: true
```
Comment (carlocab) [inline .github/workflows/automerge.yml:190]: Could probably be `false` here.

File: .github/workflows/remove-long-timeout-labels.yml [hunk 4]
```diff
@@ -82,6 +82,8 @@ jobs:
       - name: Set up Homebrew
         uses: Homebrew/actions/setup-homebrew@master
         with:
+          core: true
```
Comment (carlocab) [inline .github/workflows/remove-long-timeout-labels.yml:85]: This can be `false` here.

**Review by carlocab — APPROVED** [review]

**Review by Bo98 — APPROVED** [review]

**Pull request merged**